data.js

```diff
@@ -23,6 +23,13 @@ module.exports = [
       "Learn Programming"
     ]
   },
+  {
+    name: 'Jina AI',
+    description: 'The most advanced MLOps platform for Multimodal AI',
+    url: 'https://jina.ai',
+    twitter: '@JinaAI_',
+    tags: ['Multimodal', 'Neural Search', 'Creative AI'],
+}
```

Comment:
Hey! You need to put a comma here in line 32, after `}`

Reply:
Hey @Nick17t Do you need help regarding this PR? You can commit again and push here. Lmk if you need any help.

Reply:
thanks for the kind reminder, I submitted a new pr.